registration/include/pcl/registration/ndt.h

```diff
@@ -457,6 +457,16 @@ namespace pcl
       /** \brief The second order derivative of the transformation of a point w.r.t. the transform vector, \f$ H_E \f$ in Equation 6.20 [Magnusson 2009]. */
       Eigen::Matrix<double, 18, 6> point_hessian_;
 
+      /** \brief Will keep a copy of the Hessian \f$ H \f$ in Equation 6.13 [Magnusson 2009]. */
+      Eigen::Matrix<double, 6, 6> hessian_;
+
+    public:
```

Comment:
There's a public scope defined just below on this file. There's no need to redefine it here again.